tools/firely-server-import.rst

```diff
@@ -0,0 +1,182 @@
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --typeBuffer <typeBuffer>                 | workflow/typeBufferSize          |          | Number of resources to buffer for adding type information.                                                                                 |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --indexPar <indexPar>                     | workflow/indexParallel           |          | Number of threads to index the search parameters. This is typically the most resource intensive step and should have the most threads.     |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --indexBuffer <indexBuffer>               | workflow/indexBufferSize         |          | Number of resources to buffer for indexing the search parameters.                                                                          |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| --version                                 |                                  |          | Show version information                                                                                                                   |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+| -?, -h, --help                            |                                  |          | Show help and usage information                                                                                                            |
++-------------------------------------------+----------------------------------+----------+--------------------------------------------------------------------------------------------------------------------------------------------+
+
+Examples
+--------
+
+Runs the import for files located in directory **/path/to/your/input/files** using license file **/path/to/your/license/fsi-license.json** targeting the database defined by the connection string. In case if a resource being imported already exists in the target database, skip it.
```

Comment:
Here you say it correctly: skip it.

Reply:
😳

Reply:
I'm not sure what you meant but I feel emberresd